rtl/src/main/scala/tensil/tcu/instruction/DataMove.scala

```diff
@@ -52,12 +52,46 @@ object DataMoveArgs {
 }
 
 class DataMoveFlags extends Bundle {
-  val dataFlowControl = UInt(4.W)
+  import DataMoveKind.DataMoveKind
+  val kind = DataMoveKind(UInt(4.W))
 }
 
 object DataMoveFlags {
-  def apply(dataFlowControl: UInt): DataMoveFlags =
+  def apply(kind: UInt): DataMoveFlags =
     (new DataMoveFlags).Lit(
-      _.dataFlowControl -> dataFlowControl
+      _.kind -> kind
     )
 }
+
+object DataMoveKind {
+  type DataMoveKind = UInt
+
+  val dram0ToMemory = 0x0.U
+  val memoryToDram0 = 0x1.U
+  val dram1ToMemory = 0x2.U
+  val memoryToDram1 = 0x3.U
+  // 0x4 - 0xb are unused
```

Comment:
Does decoupling instruction flags and flow control mean we don't need this unused gap anymore?

Reply:
Correct, but I think it's fine to have the external and internal datamove kinds live at opposite ends of the allowable range. Also, this maintains backwards compatibility of existing binaries. I'm open to alternatives though!

Reply:
Good point. Let's keep it then.